src/app/personside/dialogpanel/sendMelding/standardTekster/StandardTekster.tsx

```diff
@@ -100,7 +100,8 @@ function velgTekst(
                     locale,
                     autofullforData.person,
                     autofullforData.kontor,
-                    autofullforData.saksbehandler
+                    autofullforData.saksbehandler,
+                    autofullforData.enhet
                 );
                 const ferdigTekst = captitalize(autofullfor(localeTekst, nokler));
                 settTekst(ferdigTekst);
```

Comment:
Ingen endring i test ? Finnes det noen test for velgTekst ?

Reply:
Det var ingen av de eksisterende testene som trengte å endres. Jeg har ikke funnet noen test for velgTekst.

Reply:
Det er lurt å lage en test da !